src/Configuration/Config.KeyPerFile/src/KeyPerFileConfigurationProvider.cs

```diff
@@ -30,12 +48,17 @@ private static string TrimNewLine(string value)
         /// Loads the docker secrets.
```

Comment:
Should this summary change to something general?

Reply:
Sure, loads the configuration values